src/commandExecutor.ts

```diff
@@ -60,7 +59,7 @@ const promiseTimeout = <T> (promise: Promise<T>, ms: number): Promise<T> => {
 const getCryptoProvider = async (): Promise<CryptoProvider> => {
   const ledgerPromise = async () => LedgerCryptoProvider(await TransportNodeHid.create())
   const trezorPromise = async () => TrezorCryptoProvider()
-  const cryptoProviderPromise = promiseAny([ledgerPromise(), trezorPromise()])
+  const cryptoProviderPromise = Promise.any([ledgerPromise(), trezorPromise()])
```

Comment:
There is a dependency in package.json:
`    "@types/promise.any": "^2.0.0",`
that we might no longer need.

Reply:
Removed in https://github.com/vacuumlabs/cardano-hw-cli/pull/127/commits/1ec5a1f98c0ccf6e6e496842c76f069ded1d8971.